blueprints/controller/index.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
+const useNativeDetector = require('../native-detector');
```

Comment:
Could we perhaps call this "edition-detector"?

Reply:
Resolved by https://github.com/emberjs/ember.js/pull/17621/commits/698e8b5325ca63334367e7befb93f2224cf51714